lib/modules/dosomething/dosomething_reportback/dosomething_reportback.query.inc

```diff
@@ -130,7 +130,7 @@ function dosomething_reportback_get_reportbacks_query($params) {
  * @param  array  $params
  * @return mixed
  */
-function dosomething_reportback_filter_flagged_reportbacks($params) {
+function dosomething_reportback_filter_flagged_reportbacks($params, $load_all) {
```

Comment:
I feel a little iffy about this `load_all` business. 
1. maybe that's because the name is confusing `include_flagged` may make more sense
2. it also seems strange to be returning the flagged results when the user shouldn't be getting them.